tuxemon/core/item.py

```diff
@@ -211,6 +211,10 @@ def heal(self, user, target):
         if target.current_hp == target.hp:
             return {"success": False}
 
+        # don't heal if fainted
```

Comment:
This is a commit that belongs to another PR.

Reply:
🤦‍♂ I branched off of my other branch lol. Thanks for catching this.